From b3356f34035a4595950eff421b921158ef9c6416 Mon Sep 17 00:00:00 2001
From: syphyr <syphyr@gmail.com>
Date: Tue, 12 Dec 2023 00:41:04 +0100
Subject: [PATCH] glslang: linking stdc++fs

---
 StandAlone/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/StandAlone/CMakeLists.txt b/StandAlone/CMakeLists.txt
index e7707524..7947d743 100644
--- a/StandAlone/CMakeLists.txt
+++ b/StandAlone/CMakeLists.txt
@@ -70,7 +70,7 @@ elseif(UNIX)
     endif()
 endif()
 
-target_link_libraries(glslang-standalone ${LIBRARIES})
+target_link_libraries(glslang-standalone ${LIBRARIES} stdc++fs)
 target_include_directories(glslang-standalone PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../External>)
 
-- 
2.17.1

