From 8c264f62edddb12c86c44b62cb082ca37403d6f5 Mon Sep 17 00:00:00 2001
From: syphyr <syphyr@gmail.com>
Date: Thu, 1 Jun 2023 22:17:01 +0200
Subject: [PATCH] Fix cpp-8 linking stdc++fs

---
 tools/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/CMakeLists.txt b/tools/CMakeLists.txt
index a93f6404..ed7038b7 100644
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -31,7 +31,7 @@ function(add_spvtools_tool)
 
   add_executable(${ARG_TARGET} ${ARG_SRCS})
   spvtools_default_compile_options(${ARG_TARGET})
-  target_link_libraries(${ARG_TARGET} PRIVATE ${ARG_LIBS})
+  target_link_libraries(${ARG_TARGET} PRIVATE ${ARG_LIBS} stdc++fs)
   target_include_directories(${ARG_TARGET} PRIVATE
     ${spirv-tools_SOURCE_DIR}
     ${spirv-tools_BINARY_DIR}
-- 
2.17.1

