From ff709ae7915e0e8f116af7769988b2db4bed126f Mon Sep 17 00:00:00 2001
From: syphyr <syphyr@gmail.com>
Date: Sun, 13 Aug 2023 21:44:22 +0200
Subject: [PATCH] glslang: linking stdc++fs

---
 StandAlone/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/StandAlone/CMakeLists.txt b/StandAlone/CMakeLists.txt
index b000cbd4..62d602e3 100644
--- a/StandAlone/CMakeLists.txt
+++ b/StandAlone/CMakeLists.txt
@@ -70,7 +70,7 @@ elseif(UNIX)
     endif()
 endif()
 
-target_link_libraries(glslang-standalone ${LIBRARIES})
+target_link_libraries(glslang-standalone ${LIBRARIES} stdc++fs)
 target_include_directories(glslang-standalone PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../External>
     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/External>)
@@ -86,7 +86,7 @@ if(ENABLE_SPVREMAPPER)
     add_executable(spirv-remap ${REMAPPER_SOURCES})
     set_property(TARGET spirv-remap PROPERTY FOLDER tools)
     glslang_set_link_args(spirv-remap)
-    target_link_libraries(spirv-remap ${LIBRARIES})
+    target_link_libraries(spirv-remap ${LIBRARIES} stdc++fs)
 endif()
 
 if(WIN32)
-- 
2.17.1

